Assets/LeapMotion/Core/Scripts/Attributes/QuickButtonAttribute.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Leap.Unity.Query;
+
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace Leap.Unity.Attributes {
+
+  public class QuickButtonAttribute : CombinablePropertyAttribute, IBeforeFieldAdditiveDrawer {
+
+    public const float PADDING_RIGHT = 12f;
+
+    public string label = "Quick Button";
+    public string methodOnPress = null;
+    public string tooltip = "";
+
+    public Type type;
+    public System.Reflection.MethodInfo method;
+
+    public QuickButtonAttribute(string buttonLabel, string methodOnPress, string tooltip = "") {
+      this.label = buttonLabel;
+      this.methodOnPress = methodOnPress;
+      this.tooltip = tooltip;
+    }
+
+    public float GetWidth() {
```

Comment:
I don't think this method is used?

Reply:
IBeforeFieldAdditiveDrawer uses it to determine the button width -- added a comment.